server/core/templates/apps.html

```diff
@@ -1,10 +1,15 @@
 (% extends "base_htmx.html" %)
 
-(% block title %)Error(% endblock %)
+(% block title %)Apps(% endblock %)
 
 (% block head %)
 (% endblock %)
 
+(% block nav %)
+	(% include "navbar.html" %)
+(% endblock %)
+
 (% block body %)
 	(( apps_partial|safe ))
-(% endblock %)
+	(% include "signout_modal.html" %)
```

Comment:
Maybe we need a template for a base_with_nav that also has the signout and things? That way something like apps can extend that? Since we'll need this navbar + signout in other views too.